remote/host/__main__.py

```diff
@@ -78,4 +97,4 @@ def letsencrypt():
 
 
 if __name__ == '__main__':
-    main('pianoforteqa')
+    main('root@51.15.239.29')
```

Comment:
Wouldn't it be better to have local configuration somewhere for IPs ?

Reply:
it's WIP :)

Reply:
I know I know, it's just to prove that I don't sleep :p